mne/preprocessing/interpolate.py

```diff
@@ -0,0 +1,64 @@
+"""Tools for interpolation data."""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@inria.fr>
+
+from itertools import chain
+import numpy as np
+
+from ..epochs import BaseEpochs
+
+
+def interpolate_mark_bads(insts, good_fraction=0., copy=True):
+    """Interpolate or mark bads consistently for a list of data.
+
+    Once called on a list of instances, the instances can be concatenated
+    as they will have the same list of bad channels.
+
+    Parameters
+    ----------
+    insts : list
+        The list of instances (Evoked, Epochs or Raw) to consider
+        for interpolation. Each instance should have marked channels.
+    good_fraction : float
+        A float between 0 and 1 that specifies the fraction of time
```

Comment:
```suggestion
        A float between 0 (default) and 1 that specifies the fraction of time
```
and then you'll need to re-PEP8-ify it because the line is probably too long